src/ut/diameter_handlers_test.cpp

```diff
@@ -253,10 +240,12 @@ class DiameterHandlersTest : public testing::Test
           .WillOnce(InvokeArgument<0>(irss));
       }
 
-      // Expect a delete to be sent to Sprout.
-      EXPECT_CALL(*_mock_http_conn, send_delete(http_path, _, body))
-        .Times(1)
-        .WillOnce(Return(http_ret_code)).RetiresOnSaturation();
+      // Expect that we ask the Sprout connection to deregister bindings
+      std::vector<std::string> default_impus = {IMPU3, IMPU};
```

Comment:
Why are the default IMPUs `IMPU3` and `IMPU`? I don't know enough in this area to really follow this. I expect it's all ther right thing to do, but potentially want to check

Reply:
They just are, I guess. The IMS subscription XML specifies that IMPU3 is the default impu of the implicit reg set that contains IMPU2. It's just an arbitrary choice by the author I think